clients/python/src/osparc/_api_studies_api.py

```diff
@@ -64,8 +75,14 @@ def __getattr__(self, name: str) -> Any:
         return super().__getattribute__(name)
 
     def create_study_job(self, study_id: str, job_inputs: JobInputs, **kwargs):
+        _job_inputs = _JobInputs.from_json(job_inputs.model_dump_json())
```

Comment:
not sure I quiet get why you are doing this.

Reply:
The problem is that there are two pydantic models (`JobInputs` and `_JobInputs`). `_JobInputs` is an automatically generated pydantic model from the openapi specs. And it is done in a strange way which is not backwards compatible with what we had in previous versions of the client. Hence, I wrap the automatically generated with more user friendly pydantic models (the `JobInputs`). So I need to convert one into the other.